### DISCUSSION TITLE
Is there any LSP which I can use for `R`.

### DISCUSSION THREAD

**Discussion by ArishAzmat** [TypeFox/monaco-languageclient#795] [Q&A]
I have found ```https://github.com/REditorSupport/languageserver``` but not sure how to get started with this.

**Comment by kaisalmen** [comment]
Hi @ArishAzmat if there is VSCode extension available for R you can have a look how the client is initialized and connected there. We provide a couple of examples here, but we are unable to do this for more languages.

A language server you can connect as we do in our examples: https://github.com/TypeFox/monaco-languageclient?tab=readme-ov-file#main-examples (JSON, Python, Groovy, Java)
